autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -650,6 +665,116 @@ def test_scale_up_down_on_rcv3_and_clb(self):
                           'count of {1}'.format(num_clb_nodes_after_scale,
                                                 self.min_servers))
 
+    @tags(speed='slow', type='rcv3', rcv3_mimic='pass')
+    def test_scale_down_after_manual_remove_node(self):
+        """
+        Verify that the RCV3 group can successfully scale down after a server
+        has been removed from the loadbalancer pool outside of autoscale.
+        """
+
+        # Capture a list of the node_ids of all nodes on the pool before doing
+        # anything.
+        initial_node_list = [n.id for n in self.rcv3_client.get_nodes_on_pool(
+            self.pool.id).entity.nodes]
+
+        # Create a group with an RCV3 loadbalancer pool and a minimum number
+        # of servers
+
+        lb_pools = [{'loadBalancerId': self.pool.id, 'type': 'RackConnectV3'}]
+
+        pool_group_resp = self._create_rcv3_group(
+            lb_list=lb_pools, group_min=0,
+            network_list=[self.rackconnect_network])
+        pool_group = pool_group_resp.entity
+
+        # Construct a policy to scale up servers
```

Comment:
If I created a group with a minimum of 2, in order to scale down after a deletion I would need to change the minimum. Since I would need to make some sort of change either way, I preferred the symmetry of scaling up then down.